doc/radosgw/config-ref.rst

```diff
@@ -416,6 +382,69 @@ instances or all radosgw-admin commands can be put into the ``[global]`` or the
 :Default: ``true``
 
 
+Garbage Collection Settings
+===========================
+
+The Ceph Object Gateway allocates storage for new and overwritten objects
+immediately.
+
+The Ceph Object Gateway purges the storage space used for deleted objects in the
+Ceph Storage cluster some time after the gateway deletes the objects from the
+bucket index. The process of purging the deleted object data from the Ceph 
+Storage cluster is known as Garbage Collection or GC.
+
+To view the queue of objects awaiting garbage collection, execute the following::
+
+  $ radosgw-admin gc list 
+
+Garbage collection is a background activity that may
+execute continuously or during times of low loads, depending upon how the
+administrator configures the Ceph Object Gateway. By default, the Ceph Object
+Gateway conducts GC operations continuously. Since GC operations are a normal
+part of Ceph Object Gateway operations, especially with object delete
+operations, objects eligible for garbage collection exist most of the time.
+
+Some workloads may temporarily or permanently outpace the rate of garbage
+collection activity. This is especially true of delete-heavy workloads, where
+many objects get stored for a short period of time and then deleted. For these
+types of workloads, administrators can increase the priority of garbage
+collection operations relative to other operations with the following
+configuration parameters.
+
+
+``rgw gc max objs``
+
+:Description: The maximum number of objects that may be handled by
+              garbage collection in one garbage collection processing cycle.
```

Comment:
rgw_gc_max_objs is actually the number of shards in the gc log. i don't think it's interesting as a tunable, and changing its value after initial deployment is likely to break things. i think we should consider removing it from the docs altogether

Reply:
is removing it from doc viable?  we know no one should change it--and they may already know of it, or see it in older doc;  maybe we should just strongly alert not to change it until a future release that permits resharding?

Reply:
as long as we backport this pr (which we should!), that would remove it from the docs for older releases as well

Reply:
I'd suggest strongly wording this setting as please do not change after initial deployment,, since the current docs until today already contains this configurable and searching for ceph-users posts will reveal some users tinkering with it, so not documenting would probably make users go for something wrong.